notebooks/nyc_taxi_years_correlation.ipynb

```diff
@@ -78,7 +78,9 @@
    "outputs": [],
    "source": [
     "taxi2016 = cudf.read_csv(\"taxi2016.csv\")\n",
-    "taxi2017 = cudf.read_csv(\"taxi2017.csv\")"
+    "taxi2016 = taxi2016.astype({\"tpep_pickup_datetime\": \"datetime64[us]\", \"tpep_dropoff_datetime\": \"datetime64[us]\"})\n",
+    "\n",
+    "taxi2017 = cudf.read_parquet(\"taxi2017.csv\")"
```

Comment:
`taxi2017 = cudf.read_parquet(\"taxi2017.csv\")`
should be 
`taxi2017 = cudf.read_parquet(\"taxi2017.parquet\")`